docs/Code_style.rst

```diff
@@ -32,3 +32,5 @@ Style guides that go a step beyond PEP8:
 Automatic rewriting of Python code: http://pypi.python.org/pypi/PythonTidy/1.22
 
 ``pep8`` might be a useful tool to check PEP8 compliance: https://github.com/jcrocholl/pep8
+
+Code style in easy-configs can be **automatically checked** using ``--check-style``, for example: ``eb --check-style ./sympy-1.3-intel-2018a-Python-2.7.14.eb``
```

Comment:
`easy-configs` -> `easyconfigs` or `easyconfig files` (cfr. https://easybuild.readthedocs.io/en/latest/Concepts_and_Terminology.html#easyconfig-files)

The `./` is not needed, so please drop that.

And actually `--check-contrib` is better (since that also checks whether SHA256 checksums are in place), see https://easybuild.readthedocs.io/en/latest/Contributing.html?highlight=check-style#code-style-review .

You can refer to that subsection here via:
```
:ref:`contributing_review_process_code_style`
```